src/modules/constants/errors.ts

```diff
@@ -1,4 +1,5 @@
 export const Errors = {
     dinoAlreadyBinded: 'dino.bind(): Already invoked',
-    routerNotRegistered: 'Express router is not registered with dino'
+    routerNotRegistered: 'Express router is not registered with dino',
+    baseUriInvalid: 'invalid dino baseUri'
```

Comment:
Please update the text to `Invalid baseUri to mount the dino app` . What do you think?